common-arcana.lic

```diff
@@ -101,12 +101,14 @@ module DRCA
     case match
     when 'Your desire to prepare this offensive spell suddenly slips away'
       pause 1
-      return prepare?(abbrev, mana, symbiosis, command)
+      return prepare?(abbrev, mana, symbiosis, command, tattoo_tm)
     when 'Something in the area interferes with your spell preparations'
       DRC.bput('rel symb', 'You release the', 'But you haven\'t') if symbiosis
       return false
     end
 
+    DRC.bput("target", 'You begin to weave mana lines') if tattoo_tm
```

Comment:
💡Thoughts on adding match strings if there isn’t anything to target? For example, critter dies or something and you no longer can see or are engaged with something targetable.

Reply:
Fixed, ty.